roomdb/sqlite/members.go

```diff
@@ -20,7 +20,7 @@ type Members struct {
 	db *sql.DB
 }
 
-func getAliases(mEntry *models.Member) []roomdb.Alias {
+func (m Members) getAliases(mEntry *models.Member) []roomdb.Alias {
```

Comment:
this change is just so that it conforms to the rest of the code-style better, yes?

Reply:
I like it, since it makes it clearer that it belongs to the members table, not the other aliases stuff.

Reply:
This change was needed because sqlite/invites.go needed the getAliases method and this is how I found how to share it from here to there